tests/unit/models/test_base_model.py

```diff
@@ -388,6 +388,48 @@ def test_from_dict_submit(self):
         actual = Request.from_dict(request)
         self.assertEqual(actual, expected)
 
+    # Note: BaseModel.from_xrpl and its overridden methods accept only camelCase or
+    # PascalCase inputs (i.e. snake_case is not accepted)
+    def test_from_xrpl_accepts_only_camel_case_inputs(self):
+        request = {
```

Comment:
What's the difference between this test case and `tx_snake_case_keys`? Just want to make sure I'm following what's going on here :)

Reply:
Could you split it into 2 tests (2 functions)?

Reply:
fixed in [58ff257](https://github.com/XRPLF/xrpl-py/pull/694/commits/58ff257dd58e93670efff47568cf71b8bcfc06c8)